reference/forms/types/options/form_attr.rst.inc

```diff
@@ -0,0 +1,26 @@
+``form_attr``
+~~~~~~~~~~~~~
+
+**type**: ``boolean`` or ``string`` **default**: ``false``
+
+When ``true`` and used on a form element, it adds a `"form" attribute`_. to its HTML field representation with
+its HTML form id. By doing this, form element can be rendered outside HTML form while still working as expected::
+
+    $builder->add('body', TextareaType::class, [
+        'form_attr' => true,
+    ]);
+
+This can be useful when you need to solve nested form problems.
+You can also set this to ``true`` on a root form to automatically set the "form" attribute on all its children.
+
+.. note::
+
+    When root form has no ID, ``form_attr`` required to be a string identifier to be used as form ID.
```

Comment:
When **the** root form [...] `form_attr` **is** required [...] as **the** form ID.